Content/Items/Gear/GearAlternatives.cs

```diff
@@ -30,12 +31,23 @@ internal class GearAlternativeGlobalItem : GlobalItem
 {
 	public override void OnCreated(Item item, ItemCreationContext context)
 	{
-		if (context is RecipeItemCreationContext && item.ModItem is null) // If crafted && a vanilla item
+		if (context is RecipeItemCreationContext) // If crafted && a vanilla item
 		{
-			if (GearAlternatives.HasGear(item.type))
+			if (item.ModItem is null && GearAlternatives.HasGear(item.type)) //Is Vanilla Item
 			{
 				item.SetDefaults(GearAlternatives.VanillaAlternativeToGear[item.type]);
 			}
+
+			if (item.ModItem is null)
+			{
+				return;
+			}
+
+			Type type = item.ModItem.GetType();
+			if (!type.IsAbstract && type.IsSubclassOf(typeof(VanillaClone)))
```

Comment:
Why is this written like this? You could just check `item.ModItem is VanillaClone` instead. `IsAbstract` will never be true since `item.ModItem` is necessarily instanced.

Reply:
Updated